### DISCUSSION TITLE
Command Simulation

### DISCUSSION THREAD

**Discussion by limbunny** [Zrips/CMI#2005] [Feature Requests]
**Description of issue or feature request:**

Ability to simulate commands before executing them to prevent damage to the server/players

---

**Cmi Version (using`/cmi version`): ** 8.6.6.0 

**Server Type (Spigot/Paperspigot/etc): **  Paperspigot #123 

**Server Version (using `/ver`): ** 1.14.3

**Comment by Zrips** [comment]
But then how would that even work? if you want to test out command which does something on a player or send message to the player, how would that even work then?
